docs/Dialects/FIRRTL/RationaleFIRRTL.md

```diff
@@ -331,8 +331,8 @@ enabled.
 
 ## Symbols and Inner Symbols
 
-Symbols and Inner Symbols are documented in the [symbol
-rationale](RationaleSymbols.md).  This section documents how symbols are used,
+Symbols and Inner Symbols are documented in the former symbol
```

Comment:
-former?

It's not former, it's just that our links are broken so can't refer to it in a way that works for markdown/github as well as website.  Rationale exists still, see: https://circt.llvm.org/docs/RationaleSymbols/ .